vnext/PropertySheets/External/Microsoft.ReactNative.CSharpApp.targets

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License.. 
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)\..\..'))\</ReactNativeWindowsDir>
```

Comment:
this should go in props

Reply:
is there any difference between props and targets other than the name?

Reply:
It's purely convention but it's important we follow "don't be weird" here :) . Usually projects do import of some props files, override whatever props you want, then import some targets files.